sdk/search/Azure.Search.Documents/src/autorest.md

```diff
@@ -77,6 +77,77 @@ directive:
   transform: $["x-ms-enum"].name = "SuggesterMode";
 ```
 
+
+### Add nullable annotations
```

Comment:
FYI, I had to do a lot of swagger workarounds.

Because most of these properties are re-defined in customization files non of the type changes but it would be nice to think about the consequence of having non-nullable properties that service can return a null value for.

Reply:
/cc @brjohnstmsft